lib/blocks.php

```diff
@@ -85,6 +63,87 @@ function gutenberg_reregister_core_block_types() {
 }
 add_action( 'init', 'gutenberg_reregister_core_block_types' );
 
+/**
+ * Complements the implementation of block type `core/social-icon`, whether it
```

Comment:
I like where you are going with this, but I think we decided to leave the old name: 
```suggestion
 * Complements the implementation of block type `core/social-link`, whether it
```

😃